install/update_91_92.php

```diff
@@ -798,6 +798,23 @@ function update91to92() {
       $DB->queryOrDie($query, "9.2 add saved searches alerts notification translation");
    }
 
+   // Manage pending status end date + comment
+   $migration->addField("glpi_tickets", "pendingenddate", "datetime");
```

Comment:
It would be better to check for field existance, to handle future manual updates